tests/test_cmd.py

```diff
@@ -0,0 +1,83 @@
+import sys
+from io import StringIO
+from unittest.mock import MagicMock
+
+from modelz.client import ModelzClient, ModelzResponse
+from modelz.cmd import console, inference
+from modelz.serde import RawSerde
+
+
+def test_inference(monkeypatch):
+    # Mocking the behavior of `ModelzClient.inference`
+    params_dict = {"key": "value"}
+    params_as_saved_in_file = b"Mocked Response" + b" saved in file"
+    output_file = "output.bin"
```

Comment:
Can use:
- tempfile
- BytesIO

Reply:
using tempfile now, but the test isn't passing - the output from console doesn't include the tempfile path, idk why.

Reply:
Different behavior on macOS. Check the CI log.